lib/miq_automation_engine/service_models/miq_ae_service_miq_request_task.rb

```diff
@@ -17,7 +17,7 @@ def message=(msg)
     end
 
     def finished(msg)
-      object_send(:update_and_notify_parent, :state => 'finished', :message => msg)
+      object_send(:update_and_notify_parent, :state => 'finished', :message => msg, :status => 'Ok')
```

Comment:
@tinaafitz Can you take a look into this?  I do not think we can assume a `status` of `Ok` just because the finished method was called.  Feels like this would create a lot of false-positive requests.